src/ducks/bar/statusBar.js

```diff
@@ -1,18 +1,48 @@
+/* global cordova */
+
+// TODO Move this module to cozy-ui
+
 import { getCssVariableValue } from 'cozy-ui/react/utils/color'
 
-export const setColor = color => {
+export const luminosity = hex => {
+  if (hex.length == 4) {
+    const r = hex.slice(1, 2)
+    const g = hex.slice(2, 3)
+    const b = hex.slice(3, 4)
+    hex = `#${r}${r}${g}${g}${b}${b}`
+  }
+  const R = parseInt(hex.slice(1, 3), 16)
+  const G = parseInt(hex.slice(3, 5), 16)
+  const B = parseInt(hex.slice(5, 7), 16)
+  return 0.21 * R + 0.72 * G + 0.07 * B
+}
+
+// At this luminosity, we switch the status bar theme to black
+const lumThresold = 220
```

Comment:
Why this particular threshold? Which color's luminosity is it?

Reply:
I switch to 127, th middle of 0-255.